### DISCUSSION TITLE
✅ 🔗 Add Link Normal and Hover Color controls to the Text Editor widget (Duplicate of #10855)

### DISCUSSION THREAD

**Discussion by pixelloop** [elementor/elementor#21594] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I have orange set as the link color but when adding a text editor in a section/container with an orange background the link is invisible. The only solution is to use CSS.

I use BeaverBuilder a lot too and their text editor and their rows (containers) have an option to set a custom link and link hover color.

This would be very useful with Elementor where you currently have to resort to CSS code. 

### Describe the solution you'd like

Under text color add two more options
1) Link color
2) Link hover color. 

### Describe alternatives you've considered

CSS code. But this means Elementor isn't a great page builder when standard features like this that other page builders have a solution for, have to be solved with code.

**Comment by nicholaszein** [comment]
Hello there @pixelloop!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.28.0! 🥳 

Now you can change the color of links inside the Text Editor widget! 💪  

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
